lib/backburner.js

```diff
@@ -73,6 +79,48 @@ Backburner.prototype = {
     }
   },
 
+  _trigger: function(eventName /* ...args */) {
+    var callbacks = this._eventCallbacks[eventName];
+    if (callbacks) {
+      var len = arguments.length;
```

Comment:
I wanted to have the same API surface as the existing `onEnd` and `onBegin` callback options, which both 2 arguments. We could just pass in an object with the two arguments, but that might be confusing. Up to you.